pkg/logs/config/integration_config.go

```diff
@@ -62,9 +61,8 @@ type LogsConfig struct {
 	Port int    // Network
 	Path string // File, Journald
 
-	IncludeUnits         []string `mapstructure:"include_units"`         // Journald
-	ExcludeUnits         []string `mapstructure:"exclude_units"`         // Journald
-	DisableNormalization bool     `mapstructure:"disable_normalization"` // Journald
```

Comment:
Not used?

Reply:
No, I forgot to remove `DisableNormalization` when I simplified the journald integration.